typescript/cli/src/commands/registry.ts

```diff
@@ -71,19 +72,34 @@ const listCommand: CommandModuleWithContext<{ type: ChainType }> = {
 /**
  * Addresses command
  */
-const addressesCommand: CommandModuleWithContext<{ name: string }> = {
+const addressesCommand: CommandModuleWithContext<{
+  name: string;
+  contract: string;
+}> = {
   command: 'addresses',
+  aliases: ['address', 'addy'],
   describe: 'Display the addresses of core Hyperlane contracts',
   builder: {
     name: {
       type: 'string',
       description: 'Chain to display addresses for',
       alias: 'chain',
     },
+    contract: {
+      type: 'string',
+      description: 'Specific contract to print addresses for',
```

Comment:
adding a bit more clarity:
```suggestion
      description: 'Specific contract name to print addresses for',
```